src/sentry/tasks/store.py

```diff
@@ -562,11 +562,18 @@ def _do_save_event(
             data = manager.get_data()
             if not isinstance(data, dict):
                 data = dict(data.items())
-            processing.event_processing_store.store(data)
+
+            if event_type == "transaction":
+                processing.transaction_processing_store.store(data)
+            else:
+                processing.event_processing_store.store(data)
```

Comment:
don't you want to write transactions to both, and events to one?

Reply:
Not trying to double write or read yet with this. Just creating the second `rc-processing-transaction` and making it look the same to the original for now, double reading/writing will come in a later PR.

Reply:
ok